ansible_runner/utils/capacity.py

```diff
@@ -21,17 +26,24 @@ def get_mem_in_bytes():
         return error
 
 
-def get_uuid():
-    try:
-        with open('/var/lib/dbus/machine-id') as f:
-            uuid = f.read()
-        return uuid.strip()
-    except FileNotFoundError:
-        try:
-            with open('/etc/machine-id') as f:
-                uuid = f.read()
-                return uuid.strip()
-        except FileNotFoundError:
-            error = ("Could not find /var/lib/dbus/machine-id or "
-                     "/etc/machine-id files, UUID undiscoverable.")
-            return error
+def generate_or_get_uuid():
+    uuid_directory = glob.glob('/tmp/node_uuid_*')  # Location TBD
```

Comment:
I'm guessing this is a temporary placeholder. Using `glob` here could pick up multiple files, which could introduce a whole host of unexpected behavior. This should be looking at one and only one very specific location.